decidim-core/app/views/decidim/notifications_settings/show.html.erb

```diff
@@ -71,6 +71,23 @@
 
     <% end %>
 
+    <% if @notifications_settings.meet_push_notifications_requirements? %>
+      <div class="push-notifications js-sw-mandatory" style="display: none;">
```

Comment:
Could we by default display this element and then only hide it if it's not supported?

Reply:
Yes, changed